src/Sentry.Maui/Internal/MauiEvents.cs

```diff
@@ -0,0 +1,159 @@
+namespace Sentry.Maui.Internal;
+
+internal static class MauiEvents
+{
+    // These are only going to get set and used by `MauiEventsBinder`
+    public static IHub Hub { private get; set; } = null!;
```

Comment:
these `null!` are scary, if this is only to be consumed by `MauiEventsBinder` how about make it a private inner class? if the file size bothers you, make it `partial` and split the file in two (as it is now)

There are two non-nullable static public properties that are null by default so lots of assumptions hoping this are called in the right order etc.

Or just partial class and throw all these members on `MauiEventsBinder` since they are just 1 liner helper methods anyway for it.